src/docfx/build/metadata/GlobalMetadata.cs

```diff
@@ -13,13 +13,13 @@ namespace Microsoft.Docs.Build
     internal class GlobalMetadata
     {
         // For v2 backward compatibility
-        public HashSet<string> ContributorsToExclude { get; set; } = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
+        public HashSet<string> ContributorsToExclude { get; private set; } = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
```

Comment:
nit: For dictionaries, `{ get; }` can also work with JSON deserialization. Thus I would prefer `{ get; }` to make it clear that JSON deserialization is reusing our dictionary, instead of create a new dictionary and overriding ours. (Note the `StringComparer` here, it has to be our own dictionary otherwise code logic may fail.

```suggestion
        public HashSet<string> ContributorsToExclude { get; } = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
```